### PR TITLE
added community contribution to jobs cli doc

### PR DESCRIPTION
**Pull request by sascala**
## What type of changes does your doc introduce?
- [X] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [X] High
- [ ] Medium

## I have completed these items:
- [X] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [X] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [X] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

**Comment by sascala** [comment]
@joel-hamill I updated the CLI docs to include the community contribution and made the changes for 1.9 as well.

File: 1.8/usage/jobs/getting-started.md [hunk 1]
```diff
@@ -144,6 +156,20 @@ To view details about your job's schedule, run:
...
 
+### Read Job logs
+
+Inspect the log for your job:
+
+```
+dcos task log --completed <job-id>
+```
+
+To get the log for a only a specific job run, use a job run ID from `dcos job history <job-id>`
```
Comment (joel-hamill) [inline 1.8/usage/jobs/getting-started.md]: Typo: `...for a specific job run...`

File: 1.9/usage/jobs/examples.md [hunk 2]
```diff
@@ -5,56 +5,92 @@ feature_maturity: preview
...
+    dcos job add <my-job>.json
+    ```
+    
+    Alternatively, add the job using the API.
+    ```bash
+    curl -X POST -H "Content-Type: application/json" -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/service/metronome/v1/jobs -d@/Users/<your-username>/<myjob>.json
+    ```
+
+# Create a Job with a Schedule via the DC/OS CLI
+**Note:** This example JSON only works when you add the job from the DC/OS CLI or the UI. Use the [example below](#schedule-with-api) to create a scheduled job via the API.
```
Comment (joel-hamill) [inline 1.9/usage/jobs/examples.md]: Line 37 says `via the DC/OS CLI`. Line 38 begins by saying the `CLI or the UI`. Please clean this up.

**Review by joel-hamill — COMMENTED** [review]
Added comment inline

**Review by joel-hamill — CHANGES_REQUESTED** [review]
Added comments inline.

File: 1.9/usage/jobs/examples.md (hunk 2 above)
Comment (sascala) [inline 1.9/usage/jobs/examples.md]: These are instructions for creating a job with a schedule from the CLI. But the note accurately states that jobs with schedules can be created with the CLI and the UI, just not the API. I could take out the mention of the UI, but then I'd be implying that you can't create scheduled jobs that way.

Comment (sascala) [inline 1.9/usage/jobs/examples.md]: I removed "via the DC/OS CLI".

**Comment by sascala** [comment]
@joel-hamill I made your changes -- thank you for the review.

**Pull request merged**